src/hooks/wagmi.ts

```diff
@@ -0,0 +1,94 @@
+import "src/assets/rainbowkit.css"; //have to do this for now due to test failures with import direct from library;
```

Comment:
this is hopefully temporarily. See open PR with rainbow team. This causes jest test failures if imported from their library, and using jest config to override module slows tests down significantly. Ideally this shouldn't even have to be imported, as their components should handle css imports.